clang/lib/CodeGen/CoverageMappingGen.cpp

```diff
@@ -2221,27 +2249,27 @@ struct CounterCoverageMappingBuilder
     extendRegion(E->getRHS());
     propagateCounts(getRegionCounter(E), E->getRHS());
 
+    if (llvm::EnableSingleByteCoverage)
```

Comment:
I think that in the future, it would be really useful to add debug output to this code. It is becoming quite complex, with all of the types of coverage...

Reply:
This is a small refactor for below and this itself will be swept later.